### DISCUSSION TITLE
Help needed opening the book

### DISCUSSION THREAD

**Discussion by karholmaxi** [lncrawl/lightnovel-crawler#1566] [General]
Hey, I am an amateur in all this I was able to get the incrawl but I can't open the book to read. what is the best windows app that I can use to open the book already saved in my computer. It keeps saying that the file is corrupted, and I don't know why.

**Comment by dipu-bd** [comment]
I use calibre: https://calibre-ebook.com/download

The corrupted file should be another issue. Can you upload a sample epub here so that I can test?
